app/views/users/_form.html.haml

```diff
@@ -0,0 +1,35 @@
+= form_for(@user) do |f|
+
+  .row
+    .col.pe-6
+      .row
+        .col.pe-2
+          = f.label :name
+        .col
+          .value= f.object.full_name
+      .row
+        .col.pe-2
+          = f.label :telephone
+        .col
+          .value= f.object.telephone
+
+      .row
+        .col.pe-2
+          = f.label :email
+        .col
+          .value= f.object.email
+    .col.pe-6
+      .row
+        .col.pe-2
+          .value= f.label :is_active, "Status"
+        .col
+          = f.check_box :is_active
```

Comment:
We should not allow users without `user:update` rights to toggle the `is_active` check, they should just see an unmodifiable check here. We have two options:
1- Split this view into an edit and a show view, and when the current user accesses a user, redirect him/her to edit or show based on permissions (see device_models for an example)
2- Use this view for both edit and show, but set the checkbox to be not-editable and hide the Save button.

The former is the cleanest option, while the latter is quicker to implement now